<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rock Paper Scissors</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout">
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>15</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>15</number>
    </property>
    <property name="spacing">
     <number>10</number>
    </property>
    <!-- Title -->
    <item>
     <widget class="QLabel" name="label_title">
      <property name="text">
       <string>âœŠ Rock Paper Scissors âœ‹</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
        <bold>true</bold>
       </font>
      </property>
     </widget>
    </item>
    <!-- Player Camera Section -->
    <item>
     <layout class="QVBoxLayout" name="playerCol">
      <property name="spacing">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="label_player_title">
        <property name="text">
         <string>ðŸŽ® YOU</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_camera">
        <property name="minimumSize">
         <size>
          <width>450</width>
          <height>380</height>
         </size>
        </property>
        <property name="text">
         <string>Ready</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <!-- Result Label -->
    <item>
     <widget class="QLabel" name="label_result">
      <property name="text">
       <string>Ready?</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
        <bold>true</bold>
       </font>
      </property>
     </widget>
    </item>
    <!-- Gesture Label -->
    <item>
     <widget class="QLabel" name="label_fingers">
      <property name="text">
       <string>Waiting for gesture...</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
     </widget>
    </item>
    <!-- Debug Label (hidden by default in code) -->
    <item>
     <widget class="QLabel" name="label_debug">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>150</height>
       </size>
      </property>
      <property name="text">
       <string>Debug View</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <!-- Invite Section -->
    <item>
     <layout class="QHBoxLayout" name="inviteLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QLineEdit" name="txtOpponent">
        <property name="placeholderText">
         <string>Opponent Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnInvite">
        <property name="text">
         <string>Invite</string>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <!-- Button Row -->
    <item>
     <layout class="QHBoxLayout" name="buttonRow">
      <property name="spacing">
       <number>15</number>
      </property>
      <item>
       <spacer name="btnSpacerLeft">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_start">
        <property name="text">
         <string>â–¶ Start Camera</string>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_stop">
        <property name="text">
         <string>â–  Stop Camera</string>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnPlayRound">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>âœ‹ Send Move (Game)</string>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #e67e22; color: white; font-weight: bold;</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="btnSpacerRight">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
